packages/paste-website/src/components/Footer.tsx

```diff
@@ -0,0 +1,146 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {themeGet} from 'styled-system';
+import {Link} from 'gatsby';
+
+import {Anchor} from '@twilio-paste/anchor';
+import {TWILIO_BLUE} from '../constants';
+import TwilioIcon from './svg/twilio-icon';
+
+const StyledFooter = styled.footer`
+  margin-top: 96px;
+  padding: 120px ${themeGet('space.space200')} 120px;
+  background-color: #f9fafb;
+`;
+
+const StyledFooterGrid = styled.div`
+  display: grid;
```

Comment:
Are you ok with a stacked fall back for IE 11 on these columns?

Reply:
I think for 0.1 I'm ok with that. We should define some guidelines around browser support for the docs site in a future sprint. I think since most of initial users will be internal and likely on macOS we'll be ok for the interim. But, knowing we are open source and likely will have visitors from all different types of browsers and operating systems, we should clearly define what we will support.